CI/circle_parallel.sh

```diff
@@ -16,14 +16,21 @@ function cleanup {
 
 trap cleanup EXIT
 
-if [ "$NODE_INDEX" = "1" ]; then
-  echo "Running node $NODE_INDEX to test 'samples.circleci' defined in pom.xml ..."
+if [ "$NODE_INDEX" = "0" ]; then
+  echo "Running node $NODE_INDEX to test 'samples.circleci.node$NODE_INDEX' defined in pom.xml ..."
+  #sudo update-java-alternatives -s java-1.7.0-openjdk-amd64
+  java -version
+
+  mvn --no-snapshot-updates --quiet verify -Psamples.circleci.node0 -Dorg.slf4j.simpleLogger.defaultLogLevel=error
+  mvn --no-snapshot-updates --quiet javadoc:javadoc -Psamples.circleci.node1 -Dorg.slf4j.simpleLogger.defaultLogLevel=error
+elif [ "$NODE_INDEX" = "1" ]; then
+  echo "Running node $NODE_INDEX to test 'samples.circleci.node$NODE_INDEX' defined in pom.xml ..."
```

Comment:
This code was moved from below up to here. Only the echoing of the node sample was updated.